<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="4" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-27T13:20:51 IST" name="RegressionSuite" finished-at="2022-09-27T13:21:31 IST" duration-ms="40297">
    <groups>
    </groups>
    <test started-at="2022-09-27T13:20:51 IST" name="ChromeTest" finished-at="2022-09-27T13:21:10 IST" duration-ms="18567">
      <class name="tests.VerifyRoomPage1">
        <test-method is-config="true" signature="openBrowser(java.lang.String)[pri:0, instance:tests.VerifyRoomPage1@31c88ec8]" started-at="2022-09-27T13:20:51 IST" name="openBrowser" finished-at="2022-09-27T13:20:55 IST" duration-ms="4112" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="createPOMObjects()[pri:0, instance:tests.VerifyRoomPage1@31c88ec8]" started-at="2022-09-27T13:20:55 IST" name="createPOMObjects" finished-at="2022-09-27T13:20:55 IST" duration-ms="128" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPOMObjects -->
        <test-method is-config="true" signature="openRoomsPage()[pri:0, instance:tests.VerifyRoomPage1@31c88ec8]" started-at="2022-09-27T13:20:55 IST" name="openRoomsPage" finished-at="2022-09-27T13:21:05 IST" duration-ms="9038" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openRoomsPage -->
        <test-method signature="verifyContactToHelpCenter()[pri:0, instance:tests.VerifyRoomPage1@31c88ec8]" started-at="2022-09-27T13:21:05 IST" name="verifyContactToHelpCenter" finished-at="2022-09-27T13:21:08 IST" duration-ms="3018" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
URL is wrong expected [https://www.messenger.com/hel] but found [https://www.messenger.com/help]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
URL is wrong expected [https://www.messenger.com/hel] but found [https://www.messenger.com/help]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at tests.VerifyRoomPage1.verifyContactToHelpCenter(VerifyRoomPage1.java:95)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyContactToHelpCenter -->
        <test-method is-config="true" signature="logOutFromApplication(org.testng.ITestResult)[pri:0, instance:tests.VerifyRoomPage1@31c88ec8]" started-at="2022-09-27T13:21:08 IST" name="logOutFromApplication" finished-at="2022-09-27T13:21:08 IST" duration-ms="620" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyContactToHelpCenter status=FAILURE method=VerifyRoomPage1.verifyContactToHelpCenter()[pri:0, instance:tests.VerifyRoomPage1@31c88ec8] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOutFromApplication -->
        <test-method is-config="true" signature="clearObjects()[pri:0, instance:tests.VerifyRoomPage1@31c88ec8]" started-at="2022-09-27T13:21:08 IST" name="clearObjects" finished-at="2022-09-27T13:21:08 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearObjects -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:tests.VerifyRoomPage1@31c88ec8]" started-at="2022-09-27T13:21:08 IST" name="closeBrowser" finished-at="2022-09-27T13:21:10 IST" duration-ms="1498" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- tests.VerifyRoomPage1 -->
    </test> <!-- ChromeTest -->
    <test started-at="2022-09-27T13:21:10 IST" name="FirefoxTest" finished-at="2022-09-27T13:21:31 IST" duration-ms="21573">
      <class name="tests.VerifyRoomPage1">
        <test-method is-config="true" signature="openBrowser(java.lang.String)[pri:0, instance:tests.VerifyRoomPage1@6aba2b86]" started-at="2022-09-27T13:21:10 IST" name="openBrowser" finished-at="2022-09-27T13:21:18 IST" duration-ms="7672" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="createPOMObjects()[pri:0, instance:tests.VerifyRoomPage1@6aba2b86]" started-at="2022-09-27T13:21:18 IST" name="createPOMObjects" finished-at="2022-09-27T13:21:18 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPOMObjects -->
        <test-method is-config="true" signature="openRoomsPage()[pri:0, instance:tests.VerifyRoomPage1@6aba2b86]" started-at="2022-09-27T13:21:18 IST" name="openRoomsPage" finished-at="2022-09-27T13:21:27 IST" duration-ms="9197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openRoomsPage -->
        <test-method signature="verifyContactToHelpCenter()[pri:0, instance:tests.VerifyRoomPage1@6aba2b86]" started-at="2022-09-27T13:21:27 IST" name="verifyContactToHelpCenter" finished-at="2022-09-27T13:21:29 IST" duration-ms="2064" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
URL is wrong expected [https://www.messenger.com/hel] but found [https://www.messenger.com/help]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
URL is wrong expected [https://www.messenger.com/hel] but found [https://www.messenger.com/help]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at tests.VerifyRoomPage1.verifyContactToHelpCenter(VerifyRoomPage1.java:95)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyContactToHelpCenter -->
        <test-method is-config="true" signature="logOutFromApplication(org.testng.ITestResult)[pri:0, instance:tests.VerifyRoomPage1@6aba2b86]" started-at="2022-09-27T13:21:29 IST" name="logOutFromApplication" finished-at="2022-09-27T13:21:29 IST" duration-ms="252" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyContactToHelpCenter status=FAILURE method=VerifyRoomPage1.verifyContactToHelpCenter()[pri:0, instance:tests.VerifyRoomPage1@6aba2b86] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOutFromApplication -->
        <test-method is-config="true" signature="clearObjects()[pri:0, instance:tests.VerifyRoomPage1@6aba2b86]" started-at="2022-09-27T13:21:29 IST" name="clearObjects" finished-at="2022-09-27T13:21:29 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearObjects -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:tests.VerifyRoomPage1@6aba2b86]" started-at="2022-09-27T13:21:29 IST" name="closeBrowser" finished-at="2022-09-27T13:21:31 IST" duration-ms="2367" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- tests.VerifyRoomPage1 -->
    </test> <!-- FirefoxTest -->
  </suite> <!-- RegressionSuite -->
</testng-results>
